examples/translation/prepare-iwslt17-multilingual.sh

```diff
@@ -92,7 +92,7 @@ for ((i=0;i<${#SRCS[@]};++i)); do
                 | sed -e 's/<seg id="[0-9]*">\s*//g' \
                 | sed -e 's/\s*<\/seg>\s*//g' \
                 | sed -e "s/\’/\'/g" \
-                > "$DATA/valid.${SRC}-${TGT}.${LANG}"
+                >> "$DATA/valid.${SRC}-${TGT}.${LANG}"
```

Comment:
Good catch! Ideally we would split these into valid, valid1, valid2, ...

Can you replace with something like:
```
for ((j=0;j<${#VALID_SET[@]};++j)); do
    FILE=${VALID_SET[j]}
    (...)
                | (...)
                > "$DATA/valid${j}.${SRC}-${TGT}.${LANG}"
```